cclib/parser/turbomoleparser.py

```diff
@@ -149,9 +149,75 @@ def extract(self, inputfile, line):
                     self.set_attribute('atomcoords', atomcoords)
                     self.set_attribute('atomnos', atomnos)
 
+        # Frequency values in aoforce.out
+        #        mode               7        8        9       10       11       12
+        #
+        #      frequency          53.33    88.32   146.85   171.70   251.75   289.44
+        #
+        #      symmetry            a        a        a        a        a        a
+        #
+        #         IR               YES      YES      YES      YES      YES      YES
+        # |dDIP/dQ|   (a.u.)     0.0002   0.0000   0.0005   0.0004   0.0000   0.0000
+        # intensity (km/mol)       0.05     0.00     0.39     0.28     0.00     0.00
+        # intensity (  %   )       0.05     0.00     0.40     0.28     0.00     0.00
+        #
+        #        RAMAN             YES      YES      YES      YES      YES      YES
+        #
+        #   1   c           x   0.00000  0.00001  0.00000 -0.01968 -0.04257  0.00001
+        #                   y  -0.08246 -0.08792  0.02675 -0.00010  0.00000  0.17930
+        #                   z   0.00001  0.00003  0.00004 -0.10350  0.11992 -0.00003
+        if line[5:14] == 'frequency':
+            if not hasattr(self, 'vibfreqs'):
+                self.vibfreqs = []
+
+            if not hasattr(self, 'vibdisps'):
+                self.vibdisps = []
+
+            vibfreqs = [float(i.replace('i', '-')) for i in line.split()[1:]]
+            self.vibfreqs.extend(vibfreqs)
+            self.skip_line(inputfile, ['b'])
+            line = next(inputfile)
+            if line[5:13] == 'symmetry':

```

Comment:
Is there a reason we need to use indices here, and not something more flexible like line.strip().startswith?

Reply:
Fair point, will change.